### DISCUSSION TITLE
[Uczymy się architektury] Spotkanie Architektura w Azure - Wykorzystanie Azure AD w aplikacjach #azure-solutions-architect

### DISCUSSION THREAD

**Discussion by maciejgos** [AzureCommunityPL/group#131] [Wydarzenia]
Cześć 🙂

Przed nami kolejne spotkanie gdzie dotkniemy architektury w Azure! Jeśli chcesz nauczyć się jak tworzyć architekturę chmurową dołącz do nas!

Każde spotkanie to nowe architektoniczne zadanie, które pozwala nam poznać dany fragment Azure - omawiamy nasze propozycje i w ten sposób zdobywamy wiedzę!

Najbliższe spotkanie: 26/11/2020 godzina 20:00
Tematyka: Wykorzystanie Azure AD w aplikacjach

Zadanie:
Połącz w bezpieczny sposób aplikację web z bazą danych (np. Cosmos DB) i pobierz informacje korzystając z tokenu użytkownika.

Elementy który warto wziąć pod uwagę to:
- App Service
- Cosmos DB
- Azure AD
- Private Endpoint (dla BARDZO chętnych)

Wynikiem zadania powinny być:
- diagram
- wycena rozwiązania
- dyskusja

Tym zadaniem chcielibyśmy usystematyzować podstawowe pojęcia związane z AAD:
1) Autoryzacja vs uwierzytelnianie (https://www.youtube.com/watch?v=xBG076ablpg)
2) Tenant, application, service principal
3) sposoby uwierzytelniania dostępne dla AAD
4) scenariusze aplikacyjne (https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-flows-app-scenario)

Link do spotkania: https://bit.ly/3mI0gm0
Kalendarz: https://evt.mx/g5eQsKRs

Poprzednie spotkania: https://github.com/AzureCommunityPL/group/tree/master/meetings/azure-solutions-architect

W razie pytań lub wątpliwości pisz do nas (@aryll , @maciejgos)

**Comment by aryll** [comment]
Zadanie z AAD na następne spotkanie warto "wyklikać" w portalu, zwłaszcza jeśli ktoś wcześniej nie miał okazji tworzyć tenanta AAD od samego początku. 
Magiczne rzeczy potrafią się dziać jak się tym "bawi" 😃
